cluster/aws/util.sh

```diff
@@ -562,6 +562,23 @@ function create-dhcp-option-set () {
   echo "Using DHCP option set ${DHCP_OPTION_SET_ID}"
 }
 
+function delete-dhcp-option-sets () {
+  if [[ -n "${DHCP_OPTION_SET_ID:-}" ]]; then
```

Comment:
Not sure why we check DHCP_OPTION_SET_ID.

Also maybe inline this function - it's a little less modularized, but it makes it easier to see that we're deleting consistently?